index.html

```diff
@@ -53,6 +53,218 @@ <h1>I'm Mariane <br/>
+                            <p>
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <div class="card-skills">
+                            <ul>
+                                <li>html</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                <div class="cards">
+                    <div>
+                        <img src="asset/snapshot4.svg" alt="screenshot" class="screenshot">
+                    </div>
+                    <div class="card-description">
+                        <div>
+                            <p class="card-title">Multi-Post Stories</p>
+                            <div class="card-bg-info">
+                                <p>Canopy</p>
+                                <ul>
+                                    <li>Back End Dev</li>
+                                    <li>2015</li>
+                                </ul>
+                            </div>
+                        </div>
+                        <div class="card-text">
+                            <p>
+                                A daily selection of privately personalized reads; no accounts or sign-ups required.
+                            </p>
+                        </div>
+                        <div class="card-skills">
+                            <ul>
+                                <li>html</li>
+                                <li>css</li>
+                                <li>javascript</li>
+                            </ul>
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section>
+            <div class="about-container">
+                <div class="about">
+                    <div>
+                        <h2>About<br/>
+                             Myself
+                        </h2>
+                    </div>
+                    <div>
+                        <p>
+                            Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                            If you like what you see and have a project you need coded, don't hestiate to contact me.
+                        </p>
+                    </div>
+                    <div>
+                        <div>
+                            <p id="about-heading">Let's Connect</p>
```

Comment:
- [ ] Kindly make sure to wrap the `LET'S CONNECT` title into the heading `hx` tag. :wink: